pyproject.toml

```diff
@@ -51,7 +51,6 @@ gitpython = ">=3.1.29"
 pytz = ">=2022.6"
 tqdm = ">=4.64.1"
 giturlparse = ">=0.10.0"
-reretry = ">=0.11.8"
```

Comment:
let's keep re-retry for some time. we need to port all our pipelines to new session. I'll write ticket for that